src/content/developers/docs/scaling/sidechains/index.md

```diff
@@ -7,29 +7,84 @@ incomplete: true
 
-## Pros and cons {#pros-and-cons}
+### Consensus algorithms {#consensus-algorithms}
+
+One of the qualities that make sidechains unique (i.e., different from Ethereum) is the consensus algorithm used. Ssidechains don't rely on Ethereum for consensus and can choose alternative consensus protocols that suit their needs. Some examples of consensus algorithms used on sidechains include:
+
+- [Proof-of-authority](https://wikipedia.org/wiki/Proof_of_authority)
+- [Delegated proof-of-stake](https://en.bitcoinwiki.org/wiki/DPoS)
+- [Byzantine fault tolerance](https://decrypt.co/resources/byzantine-fault-tolerance-what-is-it-explained). 
+
+Like Ethereum, sidechains have validating nodes that verify and process transactions, produce blocks, and store the blockchain state. Validators are also responsible for maintaining consensus across the network and securing it against malicious attacks.
+
+#### Block parameters 
+
+Ethereum places limits on [block times](/developers/docs/blocks/#block-time) (i.e., the time it takes to produce new blocks) and [block sizes](/developers/docs/blocks/#block-size) (i.e., the number of transactions in each block). Conversely, sidechains often adopt different parameters, such as faster block times and higher gas limits, to achieve high throughput, fast transactions, and low fees. 
+
+While this has some benefits, it has critical implications for network decentralization and security. Block parameters, like fast block times and big block sizes, increase the difficulty of running a full node—leaving a few "supernodes" responsible for securing the chain. In such a scenario, the possibility of validator collusion or a malicious takeover of the chain increases.  
+
+For blockchains to scale without harming decentralization, running a node must be open to everyone—not necessarily parties with specialized hardware. This is why efforts are underway to ensure everyone can [run a full node](/developers/docs/nodes-and-clients/#why-should-i-run-an-ethereum-node) on the Ethereum network. 
+
+### EVM compatibility {#evm-compatibility}
+
+What makes sidechains useful is their ability to execute contracts developed for the [Ethereum Virtual Machine (EVM)](/developers/docs/evm/). Most EVM-compatible sidechains support smart contracts [written in Solidity](/developers/docs/smart-contracts/languages/), allowing for compatibility with Ethereum Mainnet. 
+
+This means if you want to use your [dapp](/developers/docs/dapps/) on a sidechain, it's just a matter of deploying your [smart contract](/developers/docs/smart-contracts/) to this sidechain. It looks, feels, and acts just like Mainnet – you write contracts in Solidity, and interact with the chain via the Web3 API.
+
+Because sidechains are EVM-compatible, they are considered a useful scaling [scaling solution](/developers/docs/scaling/) for Ethereum-native dapps. With your dapp on a sidechain, users can enjoy lower gas fees and faster transactions, especially if Mainnet is congested. 
```

Comment:
```suggestion
Because sidechains are EVM-compatible, they are considered a useful [scaling solution](/developers/docs/scaling/) for Ethereum-native dapps. With your dapp on a sidechain, users can enjoy lower gas fees and faster transactions, especially if Mainnet is congested. 
```
extra scaling :)